src/components/SavedProvidersList/saved-providers-list.js

```diff
@@ -50,6 +50,27 @@ function printSavedProviders(providers) {
   printJSX(printPage);
 }
 
+function emailSavedProviders(providers) {
+  const byTypeName = _.groupBy(providers, provider => provider.typeName);
+  let emailBodyString = ''
+  const newLine = "%0D%0A"
+  _.forEach(byTypeName, (providers, typeName) => {
+    let providerString = ''
+    providerString = providerString.concat(typeName + newLine)
+    providers.forEach((provider) => {
+      providerString = providerString.concat(
+        newLine + provider.name + newLine +
+        "Address: " + provider.address + newLine +
+        "Website: " + provider.website + newLine +
+        "Phone: " + provider.telephone + newLine +
+        "Email: " + provider.email + newLine)
+      })
+    emailBodyString = emailBodyString.concat(providerString + newLine)
```

Comment:
If I save "ABCD-Allston/Brighton Head Start & Children’s Services" and go to print, the email subject cuts off at the "&". I think we need to URL encode the whole body string using encodeURIComponent. If you do that, no need to pre-encode newLine.